config/main.go

```diff
@@ -97,7 +97,7 @@ func initialize() {
 	options.SetDefault(Keys.Cert, "../test_data/test.cert") // default values of Cert and Key are for testing purposes only
 	options.SetDefault(Keys.Key, "../test_data/test.key")
 	options.SetDefault(Keys.OpenAPISpecPath, "./apispec/api.spec.json")
```

Comment:
Should this path also be updated?

Reply:
@kruai I had difficulty running tests if this path wasn't updated. The path gets switched back to `./bundles/bundles.yml` in `scripts/watch.sh`

Reply:
I read that wrong, sorry (blame Monday)! I updated `bundles.yml` path for the above reason. However, tests work just fine without updating API specs path.

Reply:
didn't we put a valid bundles in `test_dir`? can't we just use that and not override env vars in 2 places?

Reply:
@shdunning Yes, we did. However, the init in `subscriptions.go` always gets executed before the tests